app/views/returns/SubmittedReturnsView.scala.html

```diff
@@ -51,6 +51,10 @@
 
             <h1 id="top" class="heading-xlarge">@messages("submittedReturns.title")</h1>
 
+            @if(showInsolvencyContent && model.obligations.nonEmpty) {
```

Comment:
We want to show this message even if the user has no returns displayed, so you won't have to do the empty obligations check. For example, ETMP could have filtered out their returns and we want to give them a clue as to why they are not displayed. I just ran this by Martin and he's confirmed this is correct.